### PR TITLE
Fixes #24230 - headers should not bleed out either

### PR DESCRIPTION
**Pull request by ares**
the [original PR](https://github.com/Katello/katello/pull/6879) didn't add ellipsis for headers to the bug didn't pass QE, I'd appreciate if this gets to 3.7 too

### CODE REVIEW

**Comment by ares** [comment]
it would be great to get this into 3.7, seems as very low risk :-)

**Comment by ares** [comment]
anyone to review? it's very tiny and simple fix

**Comment by waldenraines** [comment]
> it would be great to get this into 3.7, seems as very low risk :-)

In order to get this into 3.7 we'll need to either:

1. Reopen [#20338](https://projects.theforeman.org/issues/20338) and set the release to 3.7
2. Open a new issue and point this PR at it

@jturel any preference on the above?

**Comment by ares** [comment]
I'm happy to open new issue, I'll repush in a moment

**Comment by ares** [comment]
updated to use a separate issue aligned to 3.7.0

**Comment by waldenraines** [comment]
[test katello]

**Comment by waldenraines** [comment]
[test kaetllo]

**Comment by waldenraines** [comment]
[test katello]

**Comment by waldenraines** [comment]
[test katello]

**Comment by jturel** [comment]
Thanks, I think in this case and especially considering the redmine upgrade I think a separate issue is the way to go

**Comment by ares** [comment]
Tests shouldn't be affected, note that katello tests were failing because of rubocop issues.

**Comment by ares** [comment]
all the failures seems to be related to foreign keys with katello_docker_tags, I guess that shouldn't prevent merging this

**Comment by waldenraines** [comment]
[test katello]

File: db/migrate/20171010172724_add_docker_manifest_list.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def up
       t.timestamps
     end
 
-    remove_foreign_key :katello_docker_tags, :column => :docker_manifest_id
+    #remove_foreign_key :katello_docker_tags, :column => :docker_manifest_id
```
Comment (stbenjam) [inline db/migrate/20171010172724_add_docker_manifest_list.rb]: What's this commented out?

Comment (waldenraines) [reply db/migrate/20171010172724_add_docker_manifest_list.rb]: Hah, good question, I didn't see this before now.

Comment (ares) [reply db/migrate/20171010172724_add_docker_manifest_list.rb]: that's result of git commit -a --amend when I was changing the message with a new ticket, sorry, repushed

**Comment by waldenraines** [comment]
Thanks @ares!

**Pull request merged**